src/blocks/discord/mslash/options.js

```diff
@@ -140,8 +140,8 @@ Blockly.Blocks[blockName] = {
 
 
 Blockly.JavaScript[blockName] = function(block) {
-    const string = Blockly.JavaScript.valueToCode(block, "BOOLEAN", Blockly.JavaScript.ORDER_ATOMIC);
     const searchType = block.getFieldValue("SEARCH");
+    const string = searchType == "String" ? Blockly.JavaScript.valueToCode(block, "BOOLEAN", Blockly.JavaScript.ORDER_ATOMIC).toLowerCase() : Blockly.JavaScript.valueToCode(block, "BOOLEAN", Blockly.JavaScript.ORDER_ATOMIC);
```

Comment:
Are we sure `searchType` doesn't change value after `string` definition?

Reply:
yes, it was const before i changed it